closed/src/java.base/share/classes/com/sun/crypto/provider/NativeGaloisCounterMode.java

```diff
@@ -343,40 +357,50 @@ int encrypt(byte[] in, int inOfs, int len) {
      */
     int encryptFinal(byte[] in, int inOfs, int len, byte[] out, int outOfs)
         throws IllegalBlockSizeException, ShortBufferException {
+        byte[] keyCopy;
+        byte[] ivCopy;
+        byte[] aad;
+        int localTagLenBytes;
+        synchronized (this) {
+            localTagLenBytes = this.tagLenBytes;
+
+            if (len > MAX_BUF_SIZE - localTagLenBytes) {
```

Comment:
Missing parentheses for `MAX_BUF_SIZE - localTagLenBytes`.

Reply:
Added.